src/components/templates/PartyMap/components/RoomModal/RoomModal.scss

```diff
@@ -48,21 +48,25 @@ $spacing: 20px;
     height: fit-content;
   }
 
+  &__content {
+    flex: auto;
+    margin-left: $spacing;
+  }
+
   &__btn-enter {
     width: 100%;
     margin-top: 10px;
   }
 
   &__userlist {
-    margin: $spacing 0;
+    margin-bottom: $spacing;
   }
 
   &__description {
     @include scrollbar;
```

Comment:
[OOS]

This should've been eliminated since in `global.scss` there is:

```scss
* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
  @include scrollbar;
}
```

But somehow it got its way back through the modals
![sparkle-scrollbar-01](https://user-images.githubusercontent.com/79229621/132234551-9ba67217-abee-4809-bc34-c61adae2b302.png)

Reply:
removed all cases in scope of this PR